static/js/components/UserImage.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by anna on 4/1/16.
+ */
+
+import React from 'react';
+
+class UserImage extends React.Component{
+  render() {
+    const { imageUrl } = this.props;
+
+    return <img className="card-image" src={imageUrl}/>;
```

Comment:
we should always have an `alt` attribute on img, even if it's empty. 

in this case can we use "profile image for <username>" (similar to edX)